collections/_authors/nitrohorse.md

```diff
@@ -0,0 +1,11 @@
+---
+title: nitrohorse
+position:
+  - Team Member
```

Comment:
```suggestion
  - Team Member
  - Author
```

This is optional, if you think you'll be adding more than a couple articles in the future.